src/main.rs

```diff
@@ -869,6 +870,78 @@ fn print_results(cfg: &Config, client: &Client, bisection_result: &BisectionResu
     eprintln!("regression in {}", toolchains[*found]);
 }
 
+fn print_final_report(
+    nightly_bisection_result: &BisectionResult,
+    ci_bisection_result: &BisectionResult,
+) {
+    let BisectionResult {
+        searched: nightly_toolchains,
+        found: nightly_found,
+        ..
+    } = nightly_bisection_result;
+
+    let BisectionResult {
+        searched: ci_toolchains,
+        found: ci_found,
+        ..
+    } = ci_bisection_result;
+
+    eprintln!("");
+    eprintln!("");
+
+    eprintln!("==================================================================================");
+    eprintln!("= Please open an issue on Rust's github repository                               =");
+    eprintln!("= https://github.com/rust-lang/rust/issues/new                                   =");
+    eprintln!("= Below you will find a text that would serve as a starting point of your report =");
+    eprintln!("==================================================================================");
+
+    eprintln!("");
+
+    eprintln!("# Regression found in the compiler");
+    eprintln!("");
+
+    eprintln!(
+        "searched nightlies: from {} to {}",
+        nightly_toolchains.first().unwrap(),
+        nightly_toolchains.last().unwrap(),
+    );
+
+    eprintln!(
+        "regressed nightly: {}",
+        nightly_toolchains[*nightly_found],
+    );
+
+    eprintln!(
+        "searched commits: from https://github.com/rust-lang/rust/commit/{} to https://github.com/rust-lang/rust/commit/{1}",
+        ci_toolchains.first().unwrap(),
+        ci_toolchains.last().unwrap(),
+    );
+
+    eprintln!(
+        "regressed commit: https://github.com/rust-lang/rust/commit/{}",
+        ci_toolchains[*ci_found],
+    );
+
+    eprintln!("source code: URL OF A REPOSITORY THAT REPRODUCES THE ERROR");
+
+    eprintln!("");
+
+    eprintln!("## Instructions");
+    eprintln!("");
+    eprintln!("PLEASE GIVE US INSTRUCTIONS ON WHAT YOUR APPLICATION DOES");
+
+    eprintln!("## Error");
+    eprintln!("");
+    eprintln!("<details><summary>COLLAPSIBLE ERROR STACKTRACE</summary>");
+    eprintln!("<p>");
+    eprintln!("");
+    eprintln!("```bash");
+    eprintln!("PASTE THE ERROR YOUR APP GIVES HERE (run with RUST_BACKTRACE=1)");
+    eprintln!("```");
```

Comment:
This section is a bit confusing -- what do we mean by "error your app gives"? I would maybe just remove it?

Reply:
I meant to paste the output of the error the (regressed) compiler is giving.